index.js

```diff
@@ -134,7 +134,7 @@ function createAgent(config) {
   if (appNames.length < 1) {
     const message =
       'New Relic requires that you name this application!\n' +
-      'Set app_name in your newrelic.js file or set environment variable\n' +
+      'Set app_name in your newrelic.js or newrelic.cjs file or set environment variable\n' +
```

Comment:
probably out of scope for this but i'm wondering if we can set some variable to the config file name because you can also specify `NEW_RELIC_CONFIG_FILENAME` so technically this message isn't truth in that case

Reply:
I was looking into how this works and `NEW_RELIC_CONFIG_FILENAME` just gives you another filename to look for *in addition to* `newrelic.js` and `newrelic.cjs`, so this message is never technically wrong.